requirements.txt

```diff
@@ -1,7 +1,7 @@
-Django==1.7.10
-httplib2==0.9.1
-feedparser==5.2.0
-Markdown==2.6.2
+Django==1.8.4
+httplib2==0.9
```

Comment:
why are all of these libraries being downgraded? (httplib2, feedparser, Markdown, cssselect, sure, python-ldap, django-appconf, django-bootstrap3, django-bootstrap-form, django-taggit, django-impersonate, django-treebeard, django-pagetree, django-pageblocks)